.github/workflows/ci.yml

```diff
@@ -58,10 +62,6 @@ jobs:
         puppet: ${{fromJson(needs.setup_matrix.outputs.puppet_major_versions)}}
     name: ${{ matrix.puppet.name }} - ${{ matrix.setfile.name }}
     steps:
-      - name: Enable IPv6 on docker
-        run: |
-          echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
-          sudo service docker restart
```

Comment:
Do you really want this to go away?

Reply:
It was removed because in some cases it lead to problems.

Reply:
We used this prefix in the past so stuff has at least one address. Without that ipv6 was completely disabled on travis I think and we copied it to github actions. But linux routing stack is a jerk and if we add this and the container want's to speak to the outside world, and the outside world has an AAAA DNS record, the docker container tries IPv6 and waits ages until it runs in a timeout. In a perfect world, cloud providers would route ipv6 prefixes to the link local address of a hypervisor and we would get a global unicast address in the containers. Sadly IPv6 support on azure/gcp/aws is meh :crying_cat_face: